lib/capabilities.js

```diff
@@ -36,7 +36,11 @@ var capabilities = {
     checks: [
       {
         check: function(cb) {
-          cb(false, 'Appmetrics does not support smart profiling.');
+          if (agent().internal.supports.watchdog) {
```

Comment:
Doesn't adapter.js do the lazy loading for us?  So appmetrics is not required/started until start() is called?